polynote-kernel/src/main/scala/polynote/config/PolynoteConfig.scala

```diff
@@ -78,6 +78,30 @@ object UI {
   implicit val decoder: Decoder[UI] = deriveDecoder
 }
 
+/**
+  * Fix polynote/polynote#613
```

Comment:
Thanks for the PR! Just a minor nitpick, I don't think we need the issue number in the comment like this :)